.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
I'm almost certain the version of JavaC that error-prone uses does not have that feature (just like it doesn't have modules/jigsaw) unfortunately.